.github/actions/setup/action.yml

```diff
@@ -3,26 +3,36 @@ name: Setup environment
 inputs:
   cache:
     description: Enable caching
-    default: "true"
+    default: 'true'
```

Comment:
Question: is there a reason to prefer single quote over double quotes for these strings? I'm happy to go either way, but we should be consistent

Reply:
I think it was my auto-prettier actually.